viewer/core/static/js/app/components/containers/app-container.js

```diff
@@ -93,8 +117,9 @@ class AppContainer extends React.Component {
 
 const mapStateToProps = function(store, ownProps) {
   return {
-    subgroups: store.subgroupsState.subgroups,
+    datasets: store.datasetState.datasets,
     metricMetadata: store.metricMetadataState.metadata,
+    subgroups: store.subgroupsState.subgroups,
```

Comment:
Actually, maybe I'll follow up with this. It's a little messy and I don't want to risk breaking the toggles.